packages/vaadin-grid/test/visual/lumo/grid.test.js

```diff
@@ -219,6 +220,49 @@ describe('grid', () => {
           await visualDiff(element, `${import.meta.url}_${dir}-sorting-single-desc`);
         });
       });
+
+      describe('row focus', () => {
+        before(async () => {
+          element = fixtureSync(`
+            <vaadin-grid style="width: 550px">
+              <vaadin-grid-column path="name.first" width="200px" flex-shrink="0" frozen></vaadin-grid-column>
+              <vaadin-grid-column path="name.last" width="200px" flex-shrink="0"></vaadin-grid-column>
+              <vaadin-grid-column path="location.city" width="200px" flex-shrink="0"></vaadin-grid-column>
+            </vaadin-grid>
+          `);
+          element.items = users;
+          flushGrid(element);
+
+          // Scroll all the way to end
+          element.$.table.scrollLeft = element.__isRTL ? -1000 : 1000;
+
+          // If supported, use tab keypress to focus the body row
+          try {
+            // Test if sendKeys is supported
+            await sendKeys({ press: 'Tab' });
+            // Switch to row focus mode
+            element.__rowFocusMode = true;
+            // Focus a header row
+            element.$.header.children[0].focus();
```

Comment:
If one more `sendKeys` is added instead, would it do the job here?

```suggestion
            await sendKeys({ press: 'Tab' });
```

Reply:
Done.